content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -27,11 +24,11 @@ Datadog's admission controller is `MutatingAdmissionWebhook` type. For more deta
 ## Configuration
 {{< tabs >}}
 {{% tab "Helm chart" %}}
```

Comment:
I noticed when I was reviewing the preview that you name the tabs differently on this page than on other pages. Normally it's just `Helm` `Operator`